test/test.js

```diff
@@ -5,15 +5,15 @@ const ASSERT = require('assert'),
   SPEC_GEN = require('../server');
 
 const PORT = 3000,
-  BASE_URL = 'http://localhost:3000/',
-  NO_URL = '?type=foo&URL=notice-that-its-in-uppercase',
-  NO_TYPE = '?url=foo&TYPE=notice-that-its-in-uppercase',
-  BAD_GENERATOR = '?type=fluxor&url=http://example.com/',
-  BAD_SHORTNAME = '?type=RESPEC&url=http://example.com/%3FshortName%3Ddiplodocus',
-  NO_RESPEC = '?type=RESPEC&url=http://example.com/',
-  SUCCESS1 = '?type=respec&url=https://w3c.github.io/manifest/',
-  SUCCESS2 = '?type=respec&url=https://w3c.github.io/payment-request/',
-  SUCCESS3 = '?type=respec&url=https://w3c.github.io/resource-hints/';
+  BASE_URL = `http://localhost:3000/?${process.env.AUTHORIZATION}`,
```

Comment:
This doesn’t look right. I think you need to urlEncode AUTHORIZATION and pass it in to the actual specs below.

Reply:
I don't understand that, @marcoscaceres. `AUTHORIZATION` contains a couple regular query parameters, which are part of the URL we want to retrieve (not part of the `url` param passed to Respec)&hellip;

Reply:
Wait. You're right. Those should be part of the `url` param.
Still, I don't understand: even your own tests running `npx` are failing now: https://travis-ci.org/w3c/spec-generator/builds/374310692?utm_source=github_status&utm_medium=notification

Reply:
The ones for the npx might need the quotes around the URL maybe?

Reply:
Closing the loop on this, "Encrypted variables are not available to untrusted builds such as pull requests coming from another repository."